include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -104,9 +104,15 @@ class CommonOperatorLoader : public ProtobufLoader {
   /// for multidirectional broadcasting.
   virtual bool hasMultidirectionalBroadcast(const llvm::StringRef typeName) = 0;
 
-  void addNodeAsOutput(const OpType &op, Node *R) {
-    for (int i = 0, e = op.output_size(); i < e; i++) {
-      nodeValueByName_[op.output(i)] = NodeValue(R, i);
+  /// Associate the name of operation outputs to a NodeValues corresponding to
+  /// node \p node. if \p numOutputs is lower than 0, then all outputs are
+  /// associated. Otherwise, the first \p numOutputs outputs are associated.
+  void addNodeAsOutput(const OpType &op, Node *node, int numOutputs = -1) {
+    numOutputs = ((numOutputs < 0) || numOutputs > op.output_size())
```

Comment:
Why do you check `numOutputs > op.output_size()` here? This seems like it would represent a bug, so wouldn't be something we'd want to silently handle/ignore.

Reply:
I was trying to avoid adding support of error handling in this function, what would require to change all calls to `addNodeAsOutput` (there are many). Would an `assert` be good enough for you ?

Reply:
Agree this seems like a bug, please use proper error handling for the loaders, most likely `RETURN_ERR_IF_NOT` in this case

Reply:
Done.